cmd/mattermost/commands/config_test.go

```diff
@@ -435,6 +437,102 @@ func TestUpdateMap(t *testing.T) {
 
 }
 
+func TestConfigMigrate(t *testing.T) {
+	th := Setup()
+	defer th.TearDown()
+
+	sqlSettings := mainHelper.GetSqlSettings()
+	sqlDSN := fmt.Sprintf("%s://%s", *sqlSettings.DriverName, *sqlSettings.DataSource)
+	fileDSN := "config.json"
+
+	ds, err := config.NewStore(sqlDSN, false)
+	require.NoError(t, err)
+	fs, err := config.NewStore(fileDSN, false)
+	require.NoError(t, err)
+
+	cases := []struct {
+		Name           string
+		method	       string
+		configStore    string
+		param          string
+		value          string
+	}{
+		{
```

Comment:
The idea of table testing is: "each entry in the table is a test case", no "each entry in the table is a step in a wider test". My point here is, it looks like you are doing 7 different checks of how it can work or fail, but actually you are doing just 2.

The list of test cases that I would like to see here would be:

"Should error without --to parameter", "Should work passing the --to and the --from", "Should work passing the --to and --config", "Should work passing --to and no --from (taking the default config file)". "Should fail passing an invalid --to" and "Should fail passing an invalid --from".

Reply:
I like @jespino's suggestion: it might even make sense not to bother with a table test, but just write the subtests explicitly:
```go
t.Run("Should error without --to parameter", func(t *testing.T) {
    // ...
})
```